src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_addresses.html.twig

```diff
@@ -1,14 +1,18 @@
-<h4 class="ui top attached styled header">
-    {{ 'sylius.ui.shipping_address'|trans }}
-</h4>
-<div class="ui attached segment" id="shipping-address">
-    {% include '@SyliusAdmin/Common/_address.html.twig' with {'address': order.shippingAddress} %}
-</div>
-<h4 class="ui attached styled header">
+{% if order.shippingAddress is not null %}
```

Comment:
Why here `if` is around whole `h4 ~ div` block, and on billing address it's only around `include`? Maybe it would be better to surround only `include` tags and provide some nice message, like:
```twig
{% if order.billingAddress is not null %}
    {% include '@SyliusAdmin/Common/_address.html.twig' with {'address': order.billingAddress} %}
{% else %}
    {{ 'sylius.ui.no_billing_address_defined'|trans }}
{% endif %}
```  
wdyt?